### PR TITLE
feat(Data Mapper): changed outputPaths to vs-code-designer/vs-code-data-mapper

### PR DESCRIPTION
**Pull request by Elaina-Lee**
- **Please check if the PR fulfills these requirements**

* [ ] The commit message follows our guidelines
* [ ] Tests for the changes have been added (for bug fixes/features)
* [ ] Docs have been added / updated (for bug fixes / features)

- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

- **What is the current behavior?** (You can also link to an open issue here)

- **What is the new behavior (if this is a feature change)?**

- **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

- **Please Include Screenshots or Videos of the intended change**:

### CODE REVIEW

File: apps/vs-code-designer/src/app/commands/dataMapper/DataMapperPanel.ts [hunk 1]
```diff
@@ -82,7 +82,7 @@ export default class DataMapperPanel {
 
   private async _setWebviewHtml() {
```
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/app/commands/dataMapper/DataMapperPanel.ts:83]: NIT: This can be set for another PR. For now its good. For this `_setWebviewHtml` function lets remove it and reuse the one from `apps/vs-code-designer/src/app/utils/codeless/getWebViewHTML.ts`.  Its basically the same code.

**Review by ccastrotrejo — APPROVED** [review]

File: apps/vs-code-designer/src/app/commands/dataMapper/DataMapperPanel.ts (hunk 1 above)
Comment (Elaina-Lee) [reply apps/vs-code-designer/src/app/commands/dataMapper/DataMapperPanel.ts:84]: I didn't notice there was the same function, thanks! I don't see other functions that could possibly exist in the helper, but if there is, I'll change it asap.

Restructured to use the function in `getWebViewHTML.ts`, but left the `_setWebviewHtml ` since that's an async function therefore the function body cannot be moved to constructor. Changed it as a async setter function instead.

File: apps/vs-code-designer/src/app/commands/dataMapper/dataMapper.ts [hunk 2]
```diff
@@ -143,7 +144,7 @@ export const loadDataMapFileCmd = async (context: IActionContext, uri: Uri) => {
       context.telemetry.properties.result = 'Canceled';
       context.telemetry.properties.missingSourceSchema = 'true';
 
-      DataMapperExt.showError('No source schema file was selected. Aborting load...');
+      ext.showError('No source schema file was selected. Aborting load...');
```
Comment (ccastrotrejo) [inline apps/vs-code-designer/src/app/commands/dataMapper/dataMapper.ts:147]: NIT: Lets localize the strings that we show in the output channel

**Review by ccastrotrejo — APPROVED** [review]

File: apps/vs-code-designer/src/app/commands/dataMapper/dataMapper.ts (hunk 2 above)
Comment (Elaina-Lee) [reply apps/vs-code-designer/src/app/commands/dataMapper/dataMapper.ts]: Localized all strings related to log statements!
